### PR TITLE
fix(Core/Spell): Val'anyr periodic healings proc and move to new proc system

### PR DESCRIPTION
**Pull request by r0m1ntik**
## Changes Proposed:
-  Move to "new" proc system

## Issues Addressed:
- Closes [4207](https://github.com/azerothcore/azerothcore-wotlk/issues/4207)

## Tests Performed:
- Tested in game (work fine)

Снимок экрана 2021-01-05 в 19 00 47

## How to Test the Changes:

1) cast heal in game (periodic or not)

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Kitzunu** [comment]
"new" proc system would be `spell_proc` but that is not fully implemented :D

File: data/sql/updates/pending_db_world/rev_1609870134525124000.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609870134525124000');
+UPDATE `spell_proc_event` SET `procFlags` = 279552, `procEx` = 786432 WHERE `entry` = 64411;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609870134525124000.sql:2]: Pls add flags like this `x`=`x`|Flag1|Flag2|Flag3 to easily see what flags have been added

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by r0m1ntik** [comment]
> "new" proc system would be `spell_proc` but that is not fully implemented :D

is empty :D

**Comment by Kitzunu** [comment]
I know :D
Btw thanks for the change =)

**Comment by pklloveyou** [comment]
Tested
image

**Comment by Kitzunu** [comment]
merge conflict

**Review by Si1ker — APPROVED** [review]

**Pull request merged**